dev/benchmarks/complex_layout/lib/main.dart

```diff
@@ -36,13 +37,45 @@ class ComplexLayoutAppState extends State<ComplexLayoutApp> {
     });
   }
 
+  ScrollMode _scrollMode = ScrollMode.complex;
+  ScrollMode get scrollMode => _scrollMode;
+  set scrollMode(ScrollMode mode) {
+    setState(() {
+      _scrollMode = mode;
+    });
+  }
+
   void toggleAnimationSpeed() {
     setState(() {
       timeDilation = (timeDilation != 1.0) ? 1.0 : 5.0;
     });
   }
 }
 
+class TileScrollLayout extends StatelessWidget {
+  const TileScrollLayout({ Key key }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final List<Widget> children = <Widget>[];
+    for (int i = 0; i < 200; i++) {
```

Comment:
Building 200 widget trees is very likely to cost many milliseconds, probably more than your frame budget. That is, building 200 widget trees, even if you do it only once, will likely cause a frame to be skipped.

You don't have to write a delegate, though. Just use the [ListView.builder](https://docs.flutter.io/flutter/widgets/ListView/ListView.builder.html) constructor and provide the builder and child count inline.